pygmt/base_plotting.py

```diff
@@ -135,6 +136,16 @@ def coast(self, **kwargs):
         shorelines : str
             ``'[level/]pen'``
             Draw shorelines [Default is no shorelines]. Append pen attributes.
+        paint_country : str or list
+            ``code1,code2,…[+l|L][+gfill][+ppen][+z]``
+            Select painting or dumping country polygons from the Digital Chart of the
+            World. Append one or more comma-separated countries using the 2-character
+            ISO 3166-1 alpha-2 convention. To select a state of a country
+            (if available), append .state, (e.g, US.TX for Texas).  To specify a whole
+            continent, prepend = to any of the continent codes (e.g. =EU for Europe).
+            Append **+p**pen to draw polygon outlines (default is no outline)
+            and **+g**fill to fill them (default is no fill). To apply different
+            settings to different countries, pass a list of string arguments.
```

Comment:
Some suggestions:

1. Add links for the "Digital Chart of the World" and "ISO 3166-1 alpha-2 convention"
2. **+z** is not explained.
3. **+l|L** is not explained.

Reply:
Changes made as suggested; I'll admit I don't have a good grasp on what **+z** and **+l** do, so it's copied from the GMT documentation.